nifi-nar-bundles/nifi-extension-utils/nifi-processor-utils/src/main/java/org/apache/nifi/processor/util/list/AbstractListProcessor.java

```diff
@@ -192,12 +192,16 @@
                     " However additional DistributedMapCache controller service is required and more JVM heap memory is used." +
                     " See the description of 'Entity Tracking Time Window' property for further details on how it works.");
 
+    public static final AllowableValue NO_TRACKING = new AllowableValue("none", "No Tracking",
+            "This strategy lists an entity without any tracking. Any property that related to persist state" +
+                    " will be disregarded.");
```

Comment:
I'd probably try to be even more descriptive here... I think it's worth saying that the same files will be listed at each execution of this processor, and that it's recommended to change the default run schedule value.

Reply:
@pvillard31 How about this: "This strategy lists an entity without any tracking. The same entity will be listed each time on executing this processor. It is recommended to change the default run schedule value. Any property that related to the persisting state will be disregarded."